subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -1,16 +1,16 @@
 #Generated file, please do not edit - Version values used in build-init templates
 commons-math=3.6.1
 commons-text=1.9
-groovy=3.0.11
+groovy=3.0.13
 guava=31.1-jre
-junit-jupiter=5.8.2
+junit-jupiter=5.9.1
 junit=4.13.2
 kotlin=1.7.10
-scala-library=2.13.8
+scala-library=2.13.9
 scala-xml=1.2.0
 scala=2.13
-scalatest=3.2.11
+scalatest=3.2.13
 scalatestplus-junit=3.2.2.0
-slf4j=1.7.36
+slf4j=2.0.2
 spock=2.1-groovy-3.0
-testng=7.5
+testng=7.6.1
```

Comment:
This will break the world.

I think you'll need to cherry-pick the changes to `UpdateInitPluginTemplateVersionFile` from 2691157f0480a9d318967a012df737b61a0558ce into `release`

Reply:
Otherwise LGTM